tests/test_parser.py

```diff
@@ -536,8 +541,5 @@ def setup_class(cls):
                NotOp(SpiresOp(Keyword('a'), Value('rodrigo,j'))))),
 
         # Dotable keys
-        ("foo.bar:baz",
-         KeywordOp(Keyword('foo.bar'), Value('baz'))),
-        ("a.b.c.d.f:bar",
-         KeywordOp(Keyword('a.b.c.d.f'), Value('bar'))),
```

Comment:
Same as the MARC keywords. If we add the "doted" keywords in the dictionary it will parse them as intended.